addon/components/bs-modal.js

```diff
@@ -193,6 +193,20 @@ export default class Modal extends Component {
   @arg
   scrollable = false;
 
+    /**
+   *  Allows adding fullscreen mode for modals. It will
+   *  apply the `modal-fullscreen' class when using 'true' and
+   *  `modal-fullscreen-[x]-down` class when using BS breakpoints
+   *   ([x] = 'sm', 'md', 'lg', 'xl', 'xxl').
+   *  
+   * @property fullscreen
+   * @type {string}
+   * @default 'true'
```

Comment:
It must not have a default value. Otherwise every modal would be fullscreen unless developer overwrites it explicitly. This would break backward compatibility.